src/components/ebay-combobox/index.marko

```diff
@@ -0,0 +1,95 @@
+$ var data = state;
```

Comment:
Let's do a bit of cleanup.

Take all `data.` attributes and change it to `input.`. Anything that changes from `onInput` put as state. (So `state.autocomplete` etc)

Reply:
Updated